english/you-are-in-minecraft/minecraft.md

```diff
@@ -0,0 +1,19 @@
+You open your eyes and there you are surrounded by a crafting table, a furnace,
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.